Example/Podfile

```diff
@@ -1,11 +1,11 @@
 use_frameworks!
-platform :ios, '8.0'
+platform :ios, '10.0'
 
 target 'Swifty_Example' do
   pod 'Swifty', :path => '../'
 
   target 'Swifty_Tests' do
-  	pod 'Alamofire', '4.7.3'
+  	pod 'Alamofire', :git => 'https://github.com/Alamofire/Alamofire.git', :tag => '5.0.0-beta.6'
```

Comment:
Should we really be testing against a beta version of Alamofire?

Reply:
For time being, until stable version is released. Any suggestions on how to test this otherwise?